docs/working-with-this-repo.md

```diff
@@ -33,7 +33,8 @@ yeoman generator in the package 'packages/generator-nitro'.
 1. Develop your new feature in the respective sample project
 2. After it works there, enhance the yeoman generator
 3. Add a test in the folder 'packages/generator-nitro/tests'
-4. Test your yeoman generator change by creating a new project
+4. Run `npm install` in the root of the repo to link to the yeoman generator with your newest changes
```

Comment:
why you think this is necessary?
should work without.

Reply:
Ah, I see. This is necessary for a new project generation.
The first time it works without